src/lib.rs

```diff
@@ -50,22 +50,40 @@ sol! {
     error Erc20InsufficientBalance(address sender, uint256 balance, uint256 needed);
 
     /// Indicates a failure with the `spender`'s `allowance`. Used in transfers.
+    /// * `spender` - address that may be allowed to operate on tokens without being their owner.
+    /// * `allowance` - amount of tokens a `spender` is allowed to operate with.
+    /// * `needed` - minimum amount required to perform a transfer.
+    error Erc20InsufficientAllowance(address sender, uint256 allowance, uint256 needed);
```

Comment:
Oh, I think we missed this in the previous PR? I remember mentioning it nit:

```suggestion
    /// * `spender` - Address that may be allowed to operate on tokens without being their owner.
    /// * `allowance` - Amount of tokens a `spender` is allowed to operate with.
    /// * `needed` - Minimum amount required to perform a transfer.
    error Erc20InsufficientAllowance(address spender, uint256 allowance, uint256 needed);
```